### PR TITLE
Feature/icons correction

### PR DESCRIPTION
**Pull request by srivastava-jyoti**

### CODE REVIEW

File: resources/views/hr/application/index.blade.php [hunk 1]
```diff
@@ -55,105 +55,90 @@
...
-                    Onboard
-                    <span
-                        class="ml-1 d-inline-block px-2 py-0 {{ $status === config('constants.hr.status.onboarded.label') ? 'active bg-white text-info' : 'bg-info text-white' }}"
-                        style="border-radius: 20px;font-size: 12px;font-weight: 700;">
-                        {{$onboardedApplicationsCount}}
-                    </span>
-                </a>
-            </li>
-        </ul>
+<div class="menu_wrapper">
```
Comment (GauravGusain98) [inline resources/views/hr/application/index.blade.php:58]: The indentation is not correct. Please fix it.

Comment (GauravGusain98) [inline resources/views/hr/application/index.blade.php:58]: The indentation is not correct. Please fix it.

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Minor changes. Please have a look

**Comment by GauravGusain98** [comment]
@srivastava-jyoti Please fix the merge conflicts and also write the description in the PR.

**Comment by srivastava-jyoti** [comment]
> @srivastava-jyoti Please fix the merge conflicts and also write the description in the PR.

@GauravGusain98 there was no issue created for this. It is a fix for the icons on the application page.

**Review by GauravGusain98 — APPROVED** [review]
Looks fine

**Pull request merged**